src/Microsoft.VisualBasic.Core/tests/Microsoft/VisualBasic/ApplicationServices/ApplicationBaseTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+namespace Microsoft.VisualBasic.ApplicationServices.Tests
+{
+    public class ApplicationBaseTests
+    {
+        [Fact]
+        public void Culture()
+        {
+            var app = new ApplicationBase();
+            var culture = app.Culture;
+            Assert.Equal(System.Threading.Thread.CurrentThread.CurrentCulture, culture);
+            try
+            {
+                app.ChangeCulture("en-US");
```

Comment:
Do we want to try setting more than just `en-US`? Does this test run on any systems where `en-US` isn't already the default.

Reply:
Same question for the `UICulture` test

---
In reply to: [277351436](https://github.com/dotnet/corefx/pull/37085#discussion_r277351436) [](ancestors = 277351436)

Reply:
My 2c: I was assuming both of these are trivial wrappers that just pass the value through, so it's not necessary (?)

Reply:
I believe the tests are run with other than `en-US`. The reason for setting `en-US` is to reduce the chance of a test failure because of missing culture info.

---
In reply to: [277351529](https://github.com/dotnet/corefx/pull/37085#discussion_r277351529) [](ancestors = 277351529,277351436)

Reply:
One of the CI legs should be es-ES (certainly the official build anyway)